### PR TITLE
Handling unicode chars

### PR DESCRIPTION
**Pull request by hunytalk**
Analytics Pipeline Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] If you have a migration please contact data engineering team before merging.
  - [ ] Before merging run full acceptance tests suite and provide URL for the acceptance tests run.
  - [x] A member of data engineering team has approved the pull request.

### CODE REVIEW

**Review by bmedx — APPROVED** [review]
Seems ok to me, just going back to replace because things downstream were breaking with unicode?

File: edx/analytics/tasks/util/encrypt.py [hunk 1]
```diff
@@ -99,14 +99,15 @@ def _import_key_files(gpg_instance, key_file_targets, hadoop_counter_incr_func=D
                             log.error("Error key with fingerprint: '%s' and recipient '%s' has expired!!!",
                                       key_fingerprint, test_key["uids"])
                             for recipient in test_key["uids"]:
-                                hadoop_counter_incr_func("GPG Key for {} has expired".format(recipient))
+                                hadoop_counter_incr_func("GPG Key for {} has expired".format(recipient.encode("ascii", "replace")))
+                                # Luigi requires that hadoop counter names be ascii-encoded, not unicode.
```
Comment (pwnage101) [inline edx/analytics/tasks/util/encrypt.py:102]: nit: can you put the comment above the line?

**Review by pwnage101 — APPROVED** [review]

**Comment by hunytalk** [comment]
We were facing hadoop counter names issue in luigi code.

File: edx/analytics/tasks/util/encrypt.py (hunk 1 above)
Comment (hunytalk) [reply edx/analytics/tasks/util/encrypt.py:102]: Sure

Comment (brianhw) [reply edx/analytics/tasks/util/encrypt.py:102]: I would also put the same comment on the change below.

**Review by pwnage101 — APPROVED** [review]

**Review by brianhw — APPROVED** [review]
One nit, but otherwise LGTM.  👍

**Pull request merged**